www/src/utils/BuildConnectionUrl.ts

```diff
@@ -0,0 +1,20 @@
+import { MineralSite } from "models";
+
+export function buildConnectionUrl(site: MineralSite, sourceConnections: Record<string, string>): string | null {
```

Comment:
do you use this function? I feel like you already have one in the SourceLink component